src/test/java/se/diabol/jenkins/pipeline/sort/NameComparatorTest.java

```diff
@@ -31,8 +31,8 @@
 
     @Test
     public void testCompare() {
-        Component componentB = new Component("B", "B", "job/A", false, new ArrayList<Pipeline>());
-        Component componentA = new Component("A", "A", "job/B", false, new ArrayList<Pipeline>());
+        Component componentB = new Component("B", "B", "job/A", false, new ArrayList<Pipeline>(), 3, false);
```

Comment:
extract false to a constant declared in the beginning of the class to declare the intent of the parameter, which then can be reused throughout the test file